supabase_py/lib/auth_client.py

```diff
@@ -7,16 +7,18 @@ class SupabaseAuthClient(gotrue.Client):
     """SupabaseAuthClient"""
 
     def __init__(
-        self,
-        auth_url: str,
-        detect_session_url: bool = False,
-        auto_refresh_token: bool = False,
-        persist_session: bool = False,
-        local_storage: Optional[Dict[str, Any]] = None,
-        headers: Dict[str, str] = {},
+            self,
+            auth_url: str,
+            detect_session_url: bool = False,
+            auto_refresh_token: bool = False,
+            persist_session: bool = False,
+            local_storage: Optional[Dict[str, Any]] = None,
+            headers = None,
```

Comment:
please rm whitespace and it appears from a glance that headers should have appropriate type of `Optional[Dict[str, Any]]`